extensions/wikia/AdEngine/js/spec/provider/gptProviders.spec.js

```diff
@@ -27,6 +27,9 @@ describe('ext.wikia.adEngine.provider.*', function () {
 			},
 			buildNew: noop
 		},
+		kiloAdUnitBuilder: {
+			build: function () {}
+		},
 		gptHelper: {
 			pushAd: function (slotName, slotElement, slotPath, slotTargeting, extra) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Function has 5 parameters which is greater than 4 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)